atomicapp/cli/main.py

```diff
@@ -37,10 +37,19 @@
 
 logger = logging.getLogger(__name__)
 
+APP_MESSAGE = """
+Your application resides in %s
+Please use this directory for managing your application
+"""
+
 
 def cli_install(args):
     try:
-        NuleculeManager.do_install(**vars(args))
+        argdict = args.__dict__
+        nm = NuleculeManager(app_spec=argdict['app_spec'],
+                             destination=argdict['destination'])
+        nm.install(**argdict)
+        print(APP_MESSAGE % nm.app_path)  # msg for users
```

Comment:
Agreed.

Perhaps when I push the logging PR that I _was_ working on there prompts / info will be a bit more __bold__.